hack/tests/e2e-go-new.sh

```diff
@@ -40,4 +40,4 @@ docker pull gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
 kind load docker-image gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
 
 trap_add cleanup EXIT
-go test -v $tests
+go test -timeout 20m -v $tests
```

Comment:
Nit/question: is this change related to this PR? If not, could we make this in a separate PR?

Reply:
It is related. OLM takes a while to install :slightly_frowning_face:.